src/app/upload/video-upload-form/video-upload-form.component.html

```diff
@@ -1,37 +1,7 @@
 <mat-card fxLayout="column">
   <bs-video-upload-progress [videoUpload]="videoUpload"></bs-video-upload-progress>
+  <div *ngIf="!this.videoSaved && !this.videoUpload.published">
+    <bs-video-form [video]="{ title: this.videoUpload.fileUpload.file.name, id: this.videoUpload.id }" [disabled]="!isPublishReady" (videoSavedEvent)="onVideoSavedEvent($event)"></bs-video-form>
```

Comment:
`{ title: this.videoUpload.fileUpload.file.name, id: this.videoUpload.id }` in html is too much, better to put it in ts